### PR TITLE
Update F39 image so that xdg-open links to flatpak-xdg-open

### LINKED ISSUE #291: xdg-open not working
This is a continuation of #147, at the moment `xdg-open` is not working as intended,

```
$ /usr/bin/xdg-open somepdf.pdf 

gio: file:somepdf.pdf: No application is registered as handling this file
```

```
/usr/bin/xdg-open "https://www.google.cl"

gio: https://www.google.cl: Operation not supported
```

The expected behaviour should be to open using the host default application, a possible fix is to create a `xdg-open` script in your path with contents
```shell
#!/bin/sh
${TOOLBOX_PATH:+flatpak-spawn --host} /usr/bin/xdg-open "$@"
```

### PR DESCRIPTION
**Pull request by notfirefox**
This implements the changes proposed [here](https://github.com/containers/toolbox/issues/291#issuecomment-1952038605).

**Changes**
- Adds the `flatpak-xdg-utils` to the list of extra packages
- Links `xdg-open` to `flatpak-xdg-open`

Closes #291

### CODE REVIEW

File: images/fedora/f39/Containerfile [hunk 1]
```diff
@@ -27,6 +27,8 @@ COPY extra-packages /
 RUN dnf -y install $(<extra-packages)
 RUN rm /extra-packages
 
+RUN ln -s /usr/bin/flatpak-xdg-open /usr/bin/xdg-open
```
Comment (debarshiray) [inline images/fedora/f39/Containerfile:30]: Nitpick: `ln --symbolic ...`, instead of `ln -s` because the long options are easier to `grep(1)`.

**Review by debarshiray — CHANGES_REQUESTED** [review]
Could you please correct your Git authorship information with your full name and a real email address, unless you are worried about privacy?

File: images/fedora/f39/Containerfile (hunk 1 above)
Comment (notfirefox) [reply images/fedora/f39/Containerfile]: I have updated the `ln` call to use the long option instead.

**Comment by notfirefox** [comment]
> Could you please correct your Git authorship information with your full name and a real email address, unless you are worried about privacy?

I'd prefer to keep the authorship information as it is because of privacy concerns.

**Comment by debarshiray** [comment]
> > Could you please correct your Git authorship information with your full name and a real email address, unless you are worried about privacy?
> 
> I'd prefer to keep the authorship information as it is because of privacy concerns.

This is blocked on the discussion in:
* https://github.com/containers/toolbox/pull/1453
* https://github.com/containers/toolbox/pull/1455

**Comment by notfirefox** [comment]
@debarshiray DCO related issues should be resolved now.

**Comment by notfirefox** [comment]
Closing in favor of https://github.com/containers/toolbox/pull/553.

**Pull request closed without merging**